test/e2e/dns.go

```diff
@@ -465,6 +465,8 @@ var _ = framework.KubeDescribe("DNS", func() {
 
 		validateTargetedProbeOutput(f, pod2, []string{wheezyFileName, jessieFileName}, "bar.example.com.")
 
+		/* Below test case is disabled for v1.7 k8s due to https://github.com/kubernetes/kubernetes/issues/35354.
+		   It is enabled for v1.8+ k8s where the fix is in place.
```

Comment:
nit: "k8s" is not so necessary here. Anyway, it's OK as-is.

Reply:
Sure, fixed the nit :)